compute/cloud-client/src/main/java/compute/reservation/CreateSharedReservation.java

```diff
@@ -57,53 +52,57 @@ public static void main(String[] args)
         "projects/%s/global/instanceTemplates/YOUR_INSTANCE_TEMPLATE_NAME", projectId);
     // Number of instances for which capacity needs to be reserved.
     int vmCount = 3;
-    // In your main method, create ReservationsClient
-    ReservationsClient client = ReservationsClient.create();
-    // Create an instance of your class, passing in the client
-    CreateSharedReservation creator = new CreateSharedReservation(client);
 
-    creator.createSharedReservation(projectId, zone, reservationName, instanceTemplateUri, vmCount);
+    createSharedReservation(projectId, zone, reservationName, instanceTemplateUri, vmCount);
   }
 
   // Creates a shared reservation with the given name in the given zone.
-  public void createSharedReservation(
-      String projectId, String zone,
-      String reservationName, String instanceTemplateUri, int vmCount)
-      throws ExecutionException, InterruptedException, TimeoutException {
+  public static Operation.Status createSharedReservation(
+          String projectId, String zone,
+          String reservationName, String instanceTemplateUri, int vmCount)
+          throws ExecutionException, InterruptedException, TimeoutException, IOException {
 
     ShareSettings shareSettings = ShareSettings.newBuilder()
```

Comment:
Move the ShareSettings within the try block as well.

Reply:
Thank you! Fixed